editor/src/clj/data_readers.clj

```diff
@@ -13,4 +13,5 @@
 ;; specific language governing permissions and limitations under the License.
 
 {code/range editor.code.data/read-cursor-range
- code/cursor editor.code.data/read-cursor}
+ code/cursor editor.code.data/read-cursor
+ g/endpoint internal.graph.types/read-endpoint}
```

Comment:
We don't use `#g/endpoint [1 :foo]` in code, but it's still nice to have support for this reader tag because we print endpoints as such. With this, we can copy REPL output with endpoints and evaluate it and get endpoints back.